src/main.rs

```diff
@@ -723,4 +725,37 @@ mod ok {
 
         results
     }
+
+    fn has_sysfs_support() -> Vec<OkTestResult> {
+        let fs_res = vec![dev_sev_rw(true), dev_sev_rw(false)];
+
+        fs_res
+    }
+
+    fn dev_sev_rw(read: bool) -> OkTestResult {
+        let path = "/dev/sev";
+        let mut success = true;
+
+        let func = if read {
+            fs::OpenOptions::new().read(true).open(path)
+        } else {
+            fs::OpenOptions::new().write(true).open(path)
+        };
+
+        let msg = match func {
+            Ok(_) => format!("{}", "Success".green()),
+            Err(e) => {
+                success = false;
+                format!("Error ({})", e).red().to_string()
+            }
+        };
+
+        let msg = if read {
+            format!("\nReading from {}: {}", path, msg)
```

Comment:
Please remove the leading `\n` in this format call. Generally when you are creating just a simple String, you should not add unnecessary punctuation or whitespace of any kind so that the caller can provide those formatting rules themselves. Furthermore, this makes it harder to track down where and why a newline is being added to the output somewhere.